pkg/plugins/golang/base/v3/plugin.go

```diff
@@ -20,10 +20,10 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
-	"sigs.k8s.io/kubebuilder/v3/pkg/plugins"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang"
 )
 
-const pluginName = "go." + plugins.DefaultNameQualifier
+const pluginName = "base." + golang.DefaultGoNameQualifier
```

Comment:
The `Go` bit is redundant
```suggestion
const pluginName = "base." + golang.DefaultNameQualifier
```

Reply:
The `Default` part could probably also be applied to `plugins.DefaultNameQualifier`

Reply:
Done and closed.